vim/vimrc.after.symlink

```diff
@@ -1,16 +0,0 @@
-color solarized
```

Comment:
Well actually it's a huge burden on the `env` to create the theme, which is why we suggest removing.